vendor/diff-parsers-std-ext.patch

```diff
@@ -535,6 +535,659 @@
+   | Pcl_extension (s, arg) ->
+-      line i ppf "Pcl_extension \"%s\"\n" s.txt;
++      line i ppf "Pcl_extension %a\n" fmt_string_loc s;
+       payload i ppf arg
+   | Pcl_open (o, e) ->
+-      line i ppf "Pcl_open %a %a\n" fmt_override_flag o.popen_override
+-        fmt_longident_loc o.popen_expr;
++      line i ppf "Pcl_open\n";
++      open_description i ppf o;
+       class_expr i ppf e
+ 
+ and class_structure i ppf { pcstr_self = p; pcstr_fields = l } =
+   line i ppf "class_structure\n";
+   pattern (i+1) ppf p;
+@@@@
+       line i ppf "Pcf_initializer\n";
+       expression (i+1) ppf e;
+   | Pcf_attribute a ->
+       attribute i ppf "Pcf_attribute" a
+   | Pcf_extension (s, arg) ->
+-      line i ppf "Pcf_extension \"%s\"\n" s.txt;
++      line i ppf "Pcf_extension %a\n" fmt_string_loc s;
+       payload i ppf arg
+ 
+ and class_field_kind i ppf = function
+   | Cfk_concrete (o, e) ->
+       line i ppf "Concrete %a\n" fmt_override_flag o;
+@@@@
+       list i with_constraint ppf l;
+   | Pmty_typeof m ->
+       line i ppf "Pmty_typeof\n";
+       module_expr i ppf m;
+   | Pmty_extension (s, arg) ->
+-      line i ppf "Pmod_extension \"%s\"\n" s.txt;
++      line i ppf "Pmod_extension %a\n" fmt_string_loc s;
+       payload i ppf arg
+ 
+ and signature i ppf x = list i signature_item ppf x
+ 
+ and signature_item i ppf x =
+@@@@
+       type_extension i ppf te
+   | Psig_exception te ->
+       line i ppf "Psig_exception\n";
+       type_exception i ppf te
+   | Psig_module pmd ->
+-      line i ppf "Psig_module %a\n" fmt_str_opt_loc pmd.pmd_name;
+-      attributes i ppf pmd.pmd_attributes;
+-      module_type i ppf pmd.pmd_type
++      line i ppf "Psig_module\n";
++      module_declaration i ppf pmd
+   | Psig_modsubst pms ->
+       line i ppf "Psig_modsubst %a = %a\n"
+         fmt_string_loc pms.pms_name
+         fmt_longident_loc pms.pms_manifest;
++      fmt_location ppf pms.pms_loc;
+       attributes i ppf pms.pms_attributes;
+   | Psig_recmodule decls ->
+       line i ppf "Psig_recmodule\n";
+       list i module_declaration ppf decls;
+   | Psig_modtype x ->
+-      line i ppf "Psig_modtype %a\n" fmt_string_loc x.pmtd_name;
+-      attributes i ppf x.pmtd_attributes;
+-      modtype_declaration i ppf x.pmtd_type
++      line i ppf "Psig_modtype\n";
++      module_type_declaration i ppf x
+   | Psig_modtypesubst x ->
+-      line i ppf "Psig_modtypesubst %a\n" fmt_string_loc x.pmtd_name;
+-      attributes i ppf x.pmtd_attributes;
+-      modtype_declaration i ppf x.pmtd_type
++      line i ppf "Psig_modtypesubst\n";
++      module_type_declaration i ppf x
+   | Psig_open od ->
+-      line i ppf "Psig_open %a %a\n" fmt_override_flag od.popen_override
+-        fmt_longident_loc od.popen_expr;
+-      attributes i ppf od.popen_attributes
++      line i ppf "Psig_open\n";
++      open_description i ppf od
+   | Psig_include incl ->
+       line i ppf "Psig_include\n";
+       module_type i ppf incl.pincl_mod;
```

Comment:
Missing `pincl_loc` (same above in signatures)

Reply:
done